index.bs

```diff
@@ -523,6 +523,37 @@ IWA frame will have access to a ControlledFrame element.
 ## Scripting methods ## {#api-scripting}
 <!-- ====================================================================== -->
 
+<xmp class="idl">
+dictionary InjectDetails {
+  DOMString? code;
+  DOMString? file;
+};
```

Comment:
Note only one can be passed per invocation.

Reply:
I added a comment above this dictionary stating that one must be specified and not both. I wasn't able to find a good way to represent this in WebIDL. The changes that I made regarding this were to appease the WebIDL checker, but this structure should not be optional for executeScript and insertCSS and it should have a required DOMString type.

I think as a future clean up of the API, perhaps this dictionary should change to something like
```
enum InjectType {
  'code',
  'file',
};

dictionary InjectDetails {
  required DOMString detail;
  required InjectType type;
};
```